metrics/web/lib/common/presentation/injector/widget/injection_container.dart

```diff
@@ -138,8 +147,13 @@ class _InjectionContainerState extends State<InjectionContainer> {
   @override
   void initState() {
     super.initState();
+    final googleSignIn = GoogleSignIn(
+      scopes: ['email'],
+      clientId: widget.metricsConfig?.googleSignInClientId,
```

Comment:
This is for release mode only as `assert` statements don't work. Just in case the given `metricsConfig` is `null`, this should prevent throwing `NoSuchMethodError` on the UI. Instead, the Google sign-in method won't work, and the UI will show an appropriate toast. Proofs:
1. Without `?.` and `null` is given (isn't even loaded due to error):
![image](https://user-images.githubusercontent.com/70939496/104703660-4863c480-5720-11eb-9420-f202fa529faf.png)
![image](https://user-images.githubusercontent.com/70939496/104704742-b52b8e80-5721-11eb-89aa-5d03289841dd.png)
2. With `?.` and `null` is given:
![image](https://user-images.githubusercontent.com/70939496/104704036-cde77480-5720-11eb-931d-283c66e59c08.png)
![image](https://user-images.githubusercontent.com/70939496/104704098-e8215280-5720-11eb-806c-d55d808a15e7.png)